packages/devextreme/testing/tests/DevExpress.ui.widgets/resizeHandle.tests.js

```diff
@@ -0,0 +1,157 @@
+
+        this.reinit();
+
+        const secondInstanceResizeStartEventName = this.instance.RESIZE_START_EVENT_NAME;
+        const secondInstanceResizeEventName = this.instance.RESIZE_EVENT_NAME;
+        const secondInstanceResizeEndEventName = this.instance.RESIZE_END_EVENT_NAME;
+
+        assert.notStrictEqual(firstInstanceResizeStartEventName, secondInstanceResizeStartEventName);
+        assert.notStrictEqual(firstInstanceResizeEventName, secondInstanceResizeEventName);
+        assert.notStrictEqual(firstInstanceResizeEndEventName, secondInstanceResizeEndEventName);
+    });
+});
+
+QUnit.module('Events', moduleConfig, () => {
+    QUnit.test('resize event handlers should be correctly added with "on" function', function(assert) {
+        const onResizeStartStub = sinon.stub();
+        const onResizeStub = sinon.stub();
+        const onResizeEndStub = sinon.stub();
+
+        this.instance.on('resize', onResizeStub);
+        this.instance.on('resizeStart', onResizeStartStub);
+        this.instance.on('resizeEnd', onResizeEndStub);
+
+        const pointer = pointerMock(this.instance.$element());
+
+        pointer.start().dragStart().drag(0, 50).drag(0, 50).dragEnd();
+
+        assert.ok(onResizeStartStub.calledOnce);
+        assert.ok(onResizeStub.calledTwice);
+        assert.ok(onResizeEndStub.calledOnce);
+    });
+
+    QUnit.test('resize event handlers should be able to update at runtime', function(assert) {
+        const onResizeStartStub = sinon.stub();
+        const onResizeStub = sinon.stub();
+        const onResizeEndStub = sinon.stub();
+
+        this.reinit({
+            onResizeStart: onResizeStartStub,
+            onResize: onResizeStub,
+            onResizeEnd: onResizeEndStub
+        });
+
+        const pointer = pointerMock(this.instance.$element());
+
+        pointer.start().dragStart().drag(0, 50).drag(0, 50).dragEnd();
+
+        assert.ok(onResizeStartStub.calledOnce);
+        assert.ok(onResizeStub.calledTwice);
+        assert.ok(onResizeEndStub.calledOnce);
+
+        const newOnResizeStartStub = sinon.stub();
+        const newOnResizeStub = sinon.stub();
+        const newOnResizeEndStub = sinon.stub();
+
+        this.instance.option('onResizeStart', newOnResizeStartStub);
+        this.instance.option('onResize', newOnResizeStub);
+        this.instance.option('onResizeEnd', newOnResizeEndStub);
+
+        pointer.start().dragStart().drag(0, 50).dragEnd();
+
+        assert.ok(newOnResizeStartStub.calledOnce);
+        assert.ok(newOnResizeStub.calledOnce);
+        assert.ok(newOnResizeEndStub.calledOnce);
+
+        assert.ok(onResizeStartStub.calledOnce);
+        assert.ok(onResizeStub.calledTwice);
+        assert.ok(onResizeEndStub.calledOnce);
+    });
+
+    QUnit.test('event handlers should recieve correct arguments', function(assert) {
+        const checkArgs = (e) => {
+            assert.ok(e.event);
```

Comment:
These asserts is to simple. We want have something like this https://github.com/EugeniyKiyashko/DevExtreme/blob/5296f968dc85e0004ac3e961feb03ddeaa7c11e7/packages/devextreme/testing/tests/DevExpress.ui.widgets.editors/dateRangeBox.tests.js#L1532